source/UnexpectedInExpected.bs

```diff
@@ -41,14 +41,18 @@ Consider the following examples (assuming `result` is of type `std::expected<int
 <tr>
     <td>
         ```cpp
-        if (!result.has_value())
-            return Error(result.error());
+        if (!result.has_value()) {
+            std::cerr << result.error() << std::endl;
```

Comment:
For brevity would it not be better to just `log_and_exit(..)` using some imaginary function?

Reply:
Then people might ask what `log_and_exit` is 😂